src/wasm-lib/kcl/src/parsing/token/mod.rs

```diff
@@ -259,6 +270,14 @@ impl<'a> winnow::stream::StreamIsPartial for TokenSlice<'a> {
     }
 }
 
+impl<'a> winnow::stream::FindSlice<&str> for TokenSlice<'a> {
```

Comment:
I had to implement this to use `take_until`. I wasn't sure if there was a better way to check and see if there was a closing bracket or not. Using `take_till` always returned Ok even if the token wasn't present